unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -495,6 +495,78 @@ public void SendSceneReady(string sceneId)
             OnReadyScene?.Invoke(sceneId);
         }
 
+        public bool TryGetScene(string id, out ParcelScene scene)
+        {
+            scene = null;
+            WorldState worldState = Environment.i.worldState;
+            if (!worldState.loadedScenes.ContainsKey(id))
+                return false;
+
+            scene = worldState.loadedScenes[id];
+            return true;
+        }
+
+        public Vector3 ConvertUnityToScenePosition(Vector3 pos, ParcelScene scene = null)
+        {
+            if (scene == null)
+            {
+                WorldState worldState = Environment.i.worldState;
+                
+                string sceneId = worldState.currentSceneId;
+
+                if (!string.IsNullOrEmpty(sceneId) && worldState.loadedScenes.ContainsKey(sceneId))
+                    scene = worldState.loadedScenes[worldState.currentSceneId];
+                else
+                    return pos;
+            }
+
+            Vector3 worldPosition = DCLCharacterController.i.characterPosition.UnityToWorldPosition(pos);
+            return worldPosition - Utils.GridToWorldPosition(scene.sceneData.basePosition.x, scene.sceneData.basePosition.y);
+        }  
+
+        public Vector3 ConvertScenePositionToUnityPosition(ParcelScene scene = null)
+        {
+            if (scene == null)
+            {
+                WorldState worldState = Environment.i.worldState;
+                string sceneId = worldState.currentSceneId;
+              
+
+                if (!string.IsNullOrEmpty(sceneId) && worldState.loadedScenes.ContainsKey(sceneId))
+                    scene = worldState.loadedScenes[worldState.currentSceneId];
+                else
+                    return Vector3.zero;
+            }
+
+            Vector3 posicionEscena = Utils.GridToWorldPosition(scene.sceneData.basePosition.x, scene.sceneData.basePosition.y);
```

Comment:
```suggestion
            Vector3 scenePosition = Utils.GridToWorldPosition(scene.sceneData.basePosition.x, scene.sceneData.basePosition.y);
```

Please don't use spanish in this repo!